### PR TITLE
Feat/pn 3695

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
create new logic for deleghe page with tab to change between two type of delegation

## List of changes proposed in this pull request
- insert tab in page
- commented useless part of deleghe page
- create new type of test, where simulate change of tab in page
- fix mobile view

## How to test
click on  deleghe in side bar menu and view title, subtitle and tab for choose which deleghe what do you want to see

### CODE REVIEW

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegatesByCompany.tsx [hunk 1]
```diff
@@ -3,7 +3,18 @@ import { useNavigate } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 import { Box, Button, Chip, Stack, Typography } from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
-import { Column, ItemsTable as Table, Item, CodeModal, Sort, EmptyState, ApiErrorWrapper } from '@pagopa-pn/pn-commons';
+import {
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegatesByCompany.tsx:6]: I should remove all the code that is useless for the purpose of the issue

File: packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx [hunk 2]
```diff
@@ -1,37 +1,45 @@
 import { useEffect, useState } from 'react';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx:1]: I should remove all te code useless for the purpose of the issue

File: packages/pn-personagiuridica-webapp/src/pages/__test__/Deleghe.page.test.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ jest.mock('../../component/Deleghe/Delegates', () => ({
   default: () => <div>delegates</div>,
 }));
 
-jest.mock('../../component/Deleghe/MobileDelegates', () => ({
+/* jest.mock('../../component/Deleghe/MobileDelegates', () => ({
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/__test__/Deleghe.page.test.tsx:25]: I should remove all the code not used

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
+import { useTranslation } from 'react-i18next';
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:1]: I should rename the file as DelegationsOfTheCompany

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  Select,
+  SelectChangeEvent,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { CustomDropdown } from '@pagopa-pn/pn-commons';
+import SearchIcon from '@mui/icons-material/Search';
+import { useState } from 'react';
+
+const ITEM_HEIGHT = 48;
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:20]: Remove the mock data and the custom style

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  'Miriam Wagner',
+  'Bradley Wilkerson',
+  'Virginia Andrews',
+  'Kelly Snyder',
+];
+
+const ProxiesChargedCompany = () => {
+  const { t } = useTranslation(['deleghe']);
+  const rows = false;
+  const [personName, setPersonName] = useState<Array<string>>([]);
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:58]: Remove the code that for now is useless. At the moment we don't need the state. Focus on the appearance of the component

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  return (
+    <>
+      <Box mb={8}>
+        <Stack mb={2} direction={'row'} justifyContent={'space-between'} alignItems={'center'}>
+          <Typography variant="h5">{t('deleghe.delegatorsTitle')}</Typography>
+        </Stack>
+        <Stack direction={'row'} spacing={2}>
+          <FormControl sx={{ width: 300 }}>
+            <InputLabel id="multiple-checkbox-label-name">Nome</InputLabel>
+            <Select
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:89]: Use textfield with attribute select

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+                <MenuItem key={name} value={name}>
+                  <Checkbox checked={personName.indexOf(name) > -1} />
+                  <ListItemText primary={name} />
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl sx={{ width: 300 }}>
+            <InputLabel id="multiple-checkbox-label-group">Gruppo</InputLabel>
+            <Select
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:112]: As before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]
Remember to write the tests for the new components

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegatesByCompany.tsx (hunk 1 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegatesByCompany.tsx:6]: done

File: packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx:1]: done

File: packages/pn-personagiuridica-webapp/src/pages/__test__/Deleghe.page.test.tsx (hunk 3 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/pages/__test__/Deleghe.page.test.tsx:25]: done

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx (hunk 4 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:1]: done

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx (hunk 5 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:20]: done

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx (hunk 6 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:58]: done

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx (hunk 7 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:89]: done

File: packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx (hunk 8 above)
Comment (leleOFA) [reply packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx:112]: done

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**